### DISCUSSION TITLE
Graph Editor

### DISCUSSION THREAD

**Discussion by sadek-hussein-yuvo** [maxGraph/maxGraph#126] [Q&A]
Hello Admin,

Regarding the new Library created with TypeScript, I see the Graph Editor is still under development, any update about it?
I can implement it. Do you have any base code already done with a specific branch? anyone can help me ?

Thanks!
![image](https://user-images.githubusercontent.com/115626918/196285835-c0d9e122-2083-4639-bab1-e7da209974e8.png)

**Comment by sadek-hussein-yuvo** [comment]
it's possible to create a view like the above one?

**Reply by tbouffard** [reply]
Probably yes, but the mxGraph examples involving the editor have not been migrated yet.
https://github.com/maxGraph/maxGraph/blob/v0.1.0/packages/html/stories/stashed/Ports.js
